### ISSUE TITLE
Please include license in tarball

### ISSUE THREAD

**Issue by sebix** [jwilk/recidivm#1]
The uploaded file recidivm-0.1.1.tar.gz does not contain the license. Please include it for the next release, thanks!

**Comment by jwilk** [comment]
Fixed in 0.1.2:

```console
$ tar -tf recidivm-0.1.2.tar.gz | grep -w LICENSE
recidivm-0.1.2/doc/LICENSE
```

**Comment by sebix** [comment]
Thanks!
